sql/parser/builtins.go

```diff
@@ -347,6 +348,80 @@ var builtins = map[string][]builtin{
 	"max": aggregateImpls(boolType, intType, floatType, stringType),
 	"min": aggregateImpls(boolType, intType, floatType, stringType),
 	"sum": aggregateImpls(intType, floatType),
+
+	// Timestamp/Date functions.
+
+	"age": {
+		builtin{
+			types:      typeList{timestampType},
+			returnType: DummyInterval,
+			fn: func(args DTuple) (Datum, error) {
+				return DInterval{Duration: time.Now().Sub(args[0].(DTimestamp).Time)}, nil
+			},
+		},
+		builtin{
+			types:      typeList{timestampType, timestampType},
+			returnType: DummyInterval,
+			fn: func(args DTuple) (Datum, error) {
+				return DInterval{Duration: args[0].(DTimestamp).Sub(args[1].(DTimestamp).Time)}, nil
+			},
+		},
+	},
+
+	"current_date": {
+		builtin{
+			types:      typeList{},
+			returnType: DummyDate,
+			fn: func(args DTuple) (Datum, error) {
+				return DDate{Time: time.Now().Truncate(24 * time.Hour)}, nil
+			},
+		},
+	},
+
+	"clock_timestamp":   {nowBuiltin()},
+	"current_timestamp": {nowBuiltin()},
+	"localtimestamp":    {nowBuiltin()},
```

Comment:
Do we need so many aliases for the same thing? `localtimestamp` is especially out of place since it is named without an underscore and we don't yet have a concept of timezones. We should at least comment which database/spec these names are coming from when there is something we can point to.